modules/sevioBidAdapter.js

```diff
@@ -14,6 +16,36 @@ const detectAdType = (bid) =>
     ["native", "banner"].find((t) => bid.mediaTypes?.[t]) || "unknown"
   ).toUpperCase();
 
+const getReferrerInfo = (bidderRequest) => {
+  return bidderRequest?.refererInfo?.page ?? '';
+}
+
+const getPageTitle = () => {
```

Comment:
is this really not on the request?

Reply:
You're right. I found the function under fpdUtils and I am using it from there now. Thanks!